book/src/using-bitfields.md

```diff
@@ -0,0 +1,196 @@
+# Using the Bitfield Types Generated by Bindgen
+
+For this discussion, we will use the following C type definitions and functions.
+```c
+typedef struct {
+    unsigned int a: 1;
+    unsigned int b: 1;
+    unsigned int c: 2;
+    
+} bitfield;
```

Comment:
I think naming the struct that contains bitfields "bitfield" leaves room for misinterpretation. The struct is not a bitfield, it just contains bitfields. How about `ContainsBitfields` or `StructWithBitfields`?

Reply:
Done